File: Daily_Codes_Answers/day-18/28_Find_the_Index_of_the_First_Occurrence_in_a_String.java
class Solution {
    static int strStr(String haystack, String needle) {
        return haystack.indexOf(needle);
    }
    public static void main(String[] args) {
        
        String haystack1 = "sadbutsad";  
        String needle1 = "sad";
        String haystack2 = "leetcode"; 
        String needle2 = "leeto";

        System.out.println("-- test case 1 --");
        System.out.println("Input : haystack - "+haystack1+" needle - "+needle1+"\nOutput : "+strStr(haystack1, needle1));

        System.out.println("-- test case 2 --");
        System.out.println("Input : haystack - "+haystack2+" needle - "+needle2+"\nOutput : "+strStr(haystack2, needle2));
    }
}

File: Daily_Codes_Answers/day-18/28_Find_the_Index_of_the_First_Occurrence_in_a_String.js
function strStr(haystack, needle) {
    let x =  haystack.indexOf(needle);
    console.log(x)
}

let haystack1 = "sadbutsad";  
let needle1 = "sad";
let haystack2 = "leetcode"; 
let needle2 = "leeto";

console.log("-- test case 1 --");
console.log("Input : haystack - "+haystack1+", needle - "+needle1+"\nOutput : "+strStr(haystack1, needle1));

console.log("-- test case 2 --");
console.log("Input : haystack - "+haystack2+", needle - "+needle2+"\nOutput : "+strStr(haystack2, needle2));
